static/styles/left-sidebar.scss

```diff
@@ -90,7 +118,7 @@ li.show-more-private-messages a {
 }
 
 #stream_filters li ul.topic-list li {
-    padding: 2px 0px 2px 29px;
```

Comment:
I see, this is dropping be 2px because of the space being dropped?  Makes sense, I think.